common/app/views/fragments/javaScriptLaterSteps.scala.html

```diff
@@ -230,7 +230,11 @@
                 'bootstraps/image-content': '@Static("javascripts/bootstraps/image-content.js")',
                 'bootstraps/membership':    '@Static("javascripts/bootstraps/membership.js")',
                 'bootstraps/sudoku':        '@Static("javascripts/bootstraps/sudoku.js")',
-                'bootstraps/video-player':  '@Static("javascripts/bootstraps/video-player.js")'
+                'bootstraps/video-player':  '@Static("javascripts/bootstraps/video-player.js")',
+                'bootstraps/article': '@Static("javascripts/bootstraps/article.js")',
+                'bootstraps/liveblog': '@Static("javascripts/bootstraps/liveblog.js")',
+                'bootstraps/trail': '@Static("javascripts/bootstraps/trail.js")',
+                'bootstraps/gallery': '@Static("javascripts/bootstraps/gallery.js")'
```

Comment:
Do you need to create jspm bundles for these too? See `bundle.js`.

RequireJS will be gone soon so we won't have to keep supporting both module loaders.